packages/utah-design-system/src/components/Sherlock.jsx

```diff
@@ -361,91 +387,102 @@ export const Sherlock = (props) => {
     props.onSherlockMatch(graphics);
   };
 
+  if (list.error) {
+    // send this to the console since we are displaying a generic error message in the UI
+    console.error(list.error);
+  }
+
   return (
-    <ComboBox
-      items={list.items}
-      inputValue={list.inputValue}
-      isLoading={list.isLoading}
-      shouldFocusWrap={true}
-      allowsEmptyCollection={true}
-      onSelectionChange={selectionChanged}
-    >
-      {props.label && <Label>{props.label}</Label>}
-      <div className="group mt-1 grow-[9999] basis-64 rounded-md transition-shadow ease-in-out">
-        <Group aria-hidden className={inputStyles}>
-          <SearchIcon
-            aria-hidden
-            className="pointer-events-none absolute inset-y-0 left-2 h-full w-5 text-zinc-400 group-focus-within:text-primary-900 dark:group-focus-within:text-zinc-300"
-          />
-          <Input
-            placeholder={props.placeholder}
-            onChange={(event) => list.setFilterText(event.target.value)}
-            className="block w-full appearance-none bg-transparent pl-9 pr-3 leading-5 text-zinc-900 caret-primary-800 placeholder:text-zinc-400 focus:outline-none dark:text-white dark:caret-accent-500 dark:ring-zinc-200/20 dark:placeholder:text-zinc-300 dark:focus:ring-accent-700 sm:text-sm"
-          />
-          {(list.loadingState === 'loading' ||
-            list.loadingState === 'filtering') && (
-            <span
-              aria-hidden
-              className="pointer-events-none text-zinc-400 group-focus-within:text-primary-900 dark:group-focus-within:text-zinc-300"
-            >
-              <Spinner ariaLabel="searching" />
-            </span>
-          )}
-          <Button className="pr-2">
-            <ChevronsUpDownIcon
+    <Form>
```

Comment:
**thought**: hopefully this won't create issues if there is a need to put sherlock inside a form. I think nested forms are ok with some caveats?

Reply:
I'm good to wait until we hit a problem before we address this.